SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security Policy
+
+## Supported Versions
+
+Only the latest version is supported.
+[Firefox might use a slightly older version](https://hg.mozilla.org/mozilla-central/file/tip/package.json#l23), which is supported-ish.
```

Comment:
Nit, the particular line where this plugin is in the mozilla-central package.json file may change at some point, maybe we could just link the file here (without referring a particular line).